BlockSettleUILib/UiUtils.cpp

```diff
@@ -612,6 +612,12 @@ QString UiUtils::NormalizeString(const QString& value)
 
 QValidator::State UiUtils::ValidateDoubleString(QString &input, int &pos, const int decimals)
 {
+   QStringList list = input.split(QLatin1Char('.'));
```

Comment:
please use defaultDecimalsSeparatorChar defined few lines below.
Also you should process "normalized" string.

Reply:
Fixed, thanks